.github/ISSUE_TEMPLATE/question.md

```diff
@@ -1,11 +1,11 @@
 ---
 name: General question
 about: Request information about the project; clarify behavior of the software
-title: '[Question]'
-labels: 'question'
+title: ''
+labels: 'kind/question'
 assignees: ''
+
 ---
 
 **How can we help?**
```

Comment:
@Ciggzy1312 will you help in switching over to `####` as opposed to `**` just for consistency?